containers/build_and_push.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+CURRENT_GIT_HASH=$(git rev-parse HEAD)
+PROJECT_ROOT=$(git rev-parse --show-toplevel)
+
+pushd $PROJECT_ROOT
+#podman pull quay.io/candlepin/subscription-manager:main
```

Comment:
When the line is commented, then there should be some explanation, why it is still present or the line should be deleted.

Reply:
Thank you!